src/main/java/at/hannibal2/skyhanni/config/features/inventory/SackDisplayConfig.java

```diff
@@ -48,7 +48,7 @@ public enum NumberFormatEntry implements HasLegacyId {
         FORMATTED("Formatted", 1),
         UNFORMATTED("Unformatted", 2);
 
-        private final String displayName;
+        public final String displayName;
```

Comment:
Generally other enum classes like this privatize `displayName` and override `toString` to return it - any reason in particular not to do the same here?

Reply:
good catch, will revert and switch to toString